src/sbf.h

```diff
@@ -65,19 +65,11 @@
 
 #define SBF_CONFIG "setSBFOutput, Stream1, %s, PVTGeodetic+VelCovGeodetic+DOP+AttEuler+AttCovEuler, msec100\n"
 
+#define SBF_CONFIG_DISABLE_OUTPUT "setDataInOut,%s%d,,-RTCMv3\n"
 
 #define SBF_CONFIG_RTCM "" \
-	"setDataInOut, USB1, Auto, RTCMv3+SBF\n" \
-	"setPVTMode, Rover, All, auto\n" \
-	"setSatelliteTracking, All\n" \
-	"setSatelliteUsage, All\n" \
-	"setElevationMask, All, 10\n" \
-	"setReceiverDynamics, Moderate, Automotive\n" \
-	"setSBFOutput, Stream1, DSK1, Support, msec100\n" \
-	"setSBFOutput, Stream2, USB1, DOP+VelCovGeodetic, sec1\n" \
-	"setSBFOutput, Stream3, USB1, PVTGeodetic, msec200\n" \
-	"setFileNaming, DSK1, Incremental\n" \
-	"setFileNaming, DSK1, , 'px4rtcm'\n"
+	"setDataInOut, USB1, Auto, RTCMv3\n" \
+	"setPVTMode, Static, All, auto\n"
```

Comment:
:+1:

Reply:
Does this also include position information in case QGC wants to know where it is?

Reply:
I looked through the source code for any use of the SBF blocks but couldn't find any. How would I verify whether QGroundControl knows the base station position? Does it normally show up on the map? I tried with some different receivers as a base station but I don't see any indication of the base station position for any of them.

Reply:
Oh ok, I thought that was how it worked but I might be dreaming :).